src/main/kotlin/com/hypto/iam/server/di/Modules.kt

```diff
@@ -124,6 +133,22 @@ val applicationModule = module {
     single { getCognitoIdentityProviderClient(get<AppConfig>().aws.region, get()) }
     single { getSesClient(get<AppConfig>().aws.region, get()) }
     single { TxMan(com.hypto.iam.server.service.DatabaseFactory.getConfiguration()) }
+    single {
+        OkHttpClient().newBuilder().apply {
+            if (get<AppConfig>().app.isDevelopment || log.isDebugEnabled) {
+                this.addInterceptor(
+                    HttpLoggingInterceptor {
+                        if (log.isDebugEnabled) {
+                            log.debug { it }
+                        } else {
+                            log.info { it }
+                        }
+                    }.setLevel(HttpLoggingInterceptor.Level.BODY)
+                )
+            }
+        }.connectionPool(ConnectionPool(MAX_IDLE_CONNECTIONS, KEEP_ALIVE_DURATION, TimeUnit.MINUTES))
+    }
+    single { get<OkHttpClient.Builder>().build() }
```

Comment:
name this client to prevent ourselves from using this for some other purpose in future

Reply:
Named